### PR TITLE
delete post following user

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Delete post following user
## Reference Requirements
- Task ID: #169

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

File: public/js/userPost.js [hunk 1]
```diff
@@ -48,3 +49,25 @@ $('#next').click(function (event) {
...
+            headers: {
+                'Accept': 'application/json',
+                'Authorization': 'Bearer ' + localStorage.getItem('access_token'),
+            },
+            success: function(data) {
+                alert("You deleted successfully!");
+                $("#"+ postId).remove();
+            },
+            error: function(data) {
+                alert(data.responseJSON.error);
```
Comment (at-vietpham) [inline public/js/userPost.js]: Hạn chế dùng `alert` show lỗi lên HTML cho người dùng.

File: public/js/userPost.js [hunk 2]
```diff
@@ -48,3 +49,25 @@ $('#next').click(function (event) {
...
+        $.ajax({
+            url: '/api/posts/' + postId,
+            type: 'DELETE',
+            headers: {
+                'Accept': 'application/json',
+                'Authorization': 'Bearer ' + localStorage.getItem('access_token'),
+            },
+            success: function(data) {
+                alert("You deleted successfully!");
+                $("#"+ postId).remove();
```
Comment (at-vietpham) [inline public/js/userPost.js]: Thêm space

File: public/js/userPost.js [hunk 3]
```diff
@@ -48,3 +49,25 @@ $('#next').click(function (event) {
...
+        confirm("Are you sure delete?");
+        $.ajax({
+            url: '/api/posts/' + postId,
+            type: 'DELETE',
+            headers: {
+                'Accept': 'application/json',
+                'Authorization': 'Bearer ' + localStorage.getItem('access_token'),
+            },
+            success: function(data) {
+                alert("You deleted successfully!");
```
Comment (at-vietpham) [inline public/js/userPost.js]: Phải load message từ file lang.

File: public/js/userPost.js [hunk 4]
```diff
@@ -48,3 +49,25 @@ $('#next').click(function (event) {
     url_next = $('#next').attr('href');
     getUserPosts(url_next);
 })
+function deletePostUser() {
+    $(document).on('click', '.delete-post-user',function() {
+        var postId = $(this).attr('id');
+        confirm("Are you sure delete?");
+        $.ajax({
+            url: '/api/posts/' + postId,
```
Comment (at-vietpham) [inline public/js/userPost.js]: Phải load message từ file lang.

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**